harness/determined/transformers/_hf_callback.py

```diff
@@ -295,8 +249,6 @@ def get_metric_type(d: Dict[str, Any]) -> str:
             return EVAL
         elif k.startswith(TEST):
             return TEST
-        else:
```

Comment:
the behavior of this function has never made sense to me. i think what you're doing is correct because if we were doing this logic for any arbitrary dict key before, then this is more or less the same behavior. think this could be expressed better though:


```python3
if any([k.startswith(EVAL) for k in d.keys()]):
    return EVAL
...
```

Reply:
yeah I like that